src/backend/magnify/core/views.py

```diff
@@ -19,7 +19,7 @@ def create_payload(user, room):
         "moderator": True,
         "aud": "jitsi",
         "iss": settings.JWT_CONFIGURATION.get("jitsi_app_id"),
-        "sub": "meet.jitsi",
+        "sub": settings.JWT_CONFIGURATION.get("jitsi_xmpp_domain"),
```

Comment:
now that it's required in settings. You'd better do that:
```suggestion
        "sub": settings.JWT_CONFIGURATION["jitsi_xmpp_domain"],
```